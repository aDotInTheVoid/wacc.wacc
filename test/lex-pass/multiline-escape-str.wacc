"\"
String not over

All of this is the same lline

\'

"
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################
########################################################################