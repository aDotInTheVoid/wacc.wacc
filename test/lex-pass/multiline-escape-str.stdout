57 string "\"
String not over

All of this is the same lline

\'

"
0 eof 
